list-navigation/lambda/handlers/record-prev-event.ts

```diff
@@ -32,8 +34,15 @@ export class RecordPrevEventHandler extends BaseApiHandler {
                 // current page tokens
                 throw new Error("No current page info in list nav session state");
             }
-
-            sessionState.upcomingPageToken = currentPageTokens.prevPageToken;
+            
+            //The last pageToken entry in the stack is used as the upcomingPageToken, in case of DDBListProvider
+            if(ListNav.getProvider(sessionState.activeList).getName() == DDBListProvider.NAME){
+                sessionState.upcomingPageToken = sessionState.pageStack?.pop();
```

Comment:
this seems especially unneeded, why would the DDB list provider not provide the tokens per the interface? (Even if it did use a stack internally)